tests/functional/object/rgw/test_rgw_http_endpoint.py

```
@@ -0,0 +1,298 @@
+        )
+
+        # Check if disableHttp parameter exists
+        ceph_object_stores = (
+            self.storage_cluster_data.get("spec", {})
+            .get("managedResources", {})
+            .get("cephObjectStores", {})
+        )
+        self.disable_http_exists = "disableHttp" in ceph_object_stores
+        self.original_disable_http = ceph_object_stores.get("disableHttp", False)
+
+        def teardown():
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `teardown`

Add return type annotation: `None`

(ANN202)